user-interface/6006-DEPC-chart_depth.md

```diff
@@ -0,0 +1,10 @@
+# Depth chart
+
+As a market user, I want to be aware of the depth of the market using a chart.
+
+- **Must** must see mid market price in the center of chart (<a name="6006-DEPC-001" href="#6006-DEPC-001">6006-DEPC-001</a>)
+- **Must** must see green bid wall and vertical axis on the left side (<a name="6006-DEPC-002" href="#6006-DEPC-002">6006-DEPC-002</a>)
+- **Must** must see red ask wall and vertical axis on the right side (<a name="6006-DEPC-003" href="#6006-DEPC-003">6006-DEPC-003</a>)
+- **Must** must see X-axis - buy or sell price (<a name="6006-DEPC-004" href="#6006-DEPC-004">6006-DEPC-004</a>)
+- **Must** must see Y-axis - value (<a name="6006-DEPC-005" href="#6006-DEPC-005">6006-DEPC-005</a>)
```

Comment:
is value the correct term here?

Is it the number of orders?

Reply:
ok I will change it to _amount of orders_